EXAMPEL.env.local

```diff
@@ -0,0 +1,8 @@
+IST_SHARED_MONGODB=
+GCP_PROJECT_ID=
+GCP_STORAGE_BUCKET=
+VERTEX_AI_ENDPOINT=
+
+MONGODB_URI=
```

Comment:
Why do we have two MongoDB connection URIs (`IST_SHARED_MONGODB` and `MONGODB_URI`)? Can we consolidate them into a single URI called `MONGODB_URI`?